pkg/auth/authenticator/password/ldappassword/ldap.go

```diff
@@ -84,17 +85,6 @@ func (a *Authenticator) getIdentity(username, password string) (authapi.UserIden
 		return nil, false, nil
 	}
 
-	// Make the connection and bind to it if a bind DN and password were given
```

Comment:
These changes shouldn't've been necessary.  Revert them.  You can make a separate pull after the main body of you changes go in.